openmetadata-ui/src/main/resources/ui/src/pages/DataModelPage/DataModelPage.component.tsx

```diff
@@ -374,9 +538,27 @@ const DataModelsPage = () => {
     }
   };
 
+  const fetchTabSpecificData = () => {
+    switch (tab) {
+      case DATA_MODELS_DETAILS_TABS.ACTIVITY: {
+        getFeedData();
+
+        break;
+      }
+
+      default:
+        break;
+    }
+  };
```

Comment:
We can remove this function and put getFeedData directly in use effect conditionally right
There's no sense of having switch case since there's only one case left